src/components/TodoFooter/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './TodoFooter';
```

Comment:
It is recommended to import the module first and then export it separately, rather than exporting directly from the import. 📚